packages/synapse-react-client/src/components/EvaluationFinder/EvaluationFinder.tsx

```diff
@@ -111,7 +111,7 @@ export default function EvaluationFinder(props: EvaluationFinderProps) {
             }
           }}
         >
-          Next
+          Next Page
```

Comment:
When shown in the Submission View wizard, the modal also has a "Next" button. Adding "Page" here so we don't have two buttons with the same text on screen at the same time